riscv_isac/cgf_normalize.py

```diff
@@ -18,7 +18,7 @@ def twos(val,bits):
     :result: two's complement version of the input
 
     '''
-    if isinstance(val,str):
+    if isinstance(val,str)
```

Comment:
Incorrect Syntax.

Reply:
This was an accidental overwrite. This line has been restored to the version in 17ff031e by rewriting the history of this branch.